contracts/ERC721B.sol

```diff
@@ -165,9 +165,9 @@ abstract contract ERC721B {
 
         // Cannot realistically overflow, since we are using uint256
         unchecked {
-            for (uint256 i = tokenId; ; i++) {
-                if (_owners[i] != address(0)) {
-                    return _owners[i];
+            for (tokenId; ; tokenId++) {
```

Comment:
uint is passed by value, not reference, so you can directly use it in a safe way here, not need to create an intermediary variable.